apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/trade/model/Event.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright © 2018-2020 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.trade.model;
+
+/**
+ * @author silaev-firstbridge on 4/14/2020
+ */
+public enum Event {
+    TRADE
```

Comment:
We have about 12 classes which have enum  Event. (Shuffling.Event, CurrencyMint.Event, ....)
So in this case I think it make sense  rename this class on EventTrade for example, to use second part of the name as class name where this event from.

Reply:
done